cmd/artifact/follow/follow.go

```diff
@@ -151,6 +153,18 @@ func NewArtifactFollowCmd(ctx context.Context, opt *options.CommonOptions) *cobr
 					o.Printer.CheckErr(fmt.Errorf("unable to overwrite \"rulesfiles-dir\" flag: %w", err))
 				}
 			}
+			// Check if directory exists and is writable
+			rulesfilesDir := fmt.Sprintf("%v",f.Value)
```

Comment:
I think here we should get the value from viper. Otherwise, when the rulesfiles dir comes from config or env we miss it. 

```
viper.Get(config.ArtifactFollowRulesfilesDirKey)
```